src/Support/Strides.hpp

```diff
@@ -0,0 +1,118 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+//===---------------------------- Strides.hpp -----------------------------===//
+//
+// Strides helper functions.
```

Comment:
Could you just add small description of what stride are supported? N dimensional? does it have to be multiple of the dimension of the original arrays, are they any restrictions...

Reply:
added